### PR TITLE
Task 608

### PR DESCRIPTION
**Pull request by john-kheir**

### CODE REVIEW

File: functional_testing/G8os/testsuite/c_advanced/tests_03_networking.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+
+    def test001_vxlans_connections(self):
+        """ g8os-029
+        *Test case for testing vxlans connections*
+
+        **Test Scenario:**
+        #. Create container (c1) on a new vxlan bridge (vx1), should succeed
+        #. Create container (c2) connected on (vx1) and connect it to default network
+        #. Create virtual machine (vm1) on (vx1), should succeed
```
Comment (ahmedelsayed-93) [inline functional_testing/G8os/testsuite/c_advanced/tests_03_networking.py]: you didn't implement this step

File: functional_testing/G8os/testsuite/c_advanced/tests_03_networking.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        rs = dhcp_c_client.system('apt-get install -y dnsmasq-base').get()
+        self.assertEqual(rs.state, 'SUCCESS')
+        dhcp_c_client.system('dnsmasq --no-hosts --keep-in-foreground --listen-address=192.168.1.1 --interface=eth0 --dhcp-range=192.168.1.2,192.168.1.3,255.255.0.0 --bind-interfaces --except-interface=lo')
+        time.sleep(40)
+
+        self.lg('Create container (c1) on a new vlan bridge (v1), should succeed')
+        nic1 = [{'type': 'vlan', 'id': v1_id, 'config': {'dhcp': True}}]
+        c1 = self.client.container.create(root_url=self.root_url, storage=self.storage, nics=nic1)
+        c1_client = self.client.container.client(c1)
+        time.sleep(6)
```
Comment (ahmedelsayed-93) [inline functional_testing/G8os/testsuite/c_advanced/tests_03_networking.py]: i'm not sure, but i think you don't have to put delay after ```self.client.container.client```

File: functional_testing/G8os/testsuite/c_advanced/tests_03_networking.py (hunk 1 above)
Comment (john-kheir) [inline functional_testing/G8os/testsuite/c_advanced/tests_03_networking.py]: i won't implement it in the scope of that test

File: functional_testing/G8os/testsuite/c_advanced/tests_03_networking.py (hunk 2 above)
Comment (john-kheir) [inline functional_testing/G8os/testsuite/c_advanced/tests_03_networking.py]: just making sure the container will take ip from the dhcp server as it sometimes took time

**Pull request merged**